k8s/kubectl/admin/configmap/wmgr/wmgr.secret

```diff
@@ -0,0 +1 @@
+GkKR0uhPxb5a:`x,/xDZq2/]Od8,jZrrde;a
```

Comment:
Secrets should not be stored in github

Reply:
It's true for the production tickets. But it may make a sense to have a file with the right name and some meaningless content such: CHANGEME

Reply:
I think the file should still be present with some meaningless content like: CHANGEME
Another possibility would be to turn it into a template which would be processed and populated at the deployment phase.

Reply:
This is a production ticket (meaning I have no idea what is a non-production ticket). So, I should insist that secrets should not appear in github 😃

Reply:
Please keep in my I'll add an automated secret generator for k8s in a short-term ticket. So secret will be generated at runtime (it wasn't possible with shmux setup which was creating secret at build time...)